Foodiea/FindUser/VC/FIndUserViewController.m

```diff
@@ -47,18 +48,27 @@ - (IBAction)didTapFilter:(id)sender {
 - (void)fetchUsers {
     PFQuery *userQuery = [PFUser query];
     [userQuery includeKey:@"author"];
+    NSLog(@"%@", self.location);
+    NSLog(@"%@", self.fav);
+    NSLog(@"%@", self.price);
```

Comment:
This is more of a nit: I would avoid comitting logs that seem just for developing at that time..

If you want logs that serve as checkpoints for debugging, that's good. If this is what it's meant to be, then I would add more information and making it one log line.